packages/expo-cli/src/commands/bundle.ts

```diff
@@ -0,0 +1,139 @@
+import { getConfig, getDefaultTarget, ProjectTarget } from '@expo/config';
+import { buildPublishBundlesAsync, shouldUseDevServer } from '@expo/xdl/build/Project';
+import { saveAssetsAsync } from '@expo/xdl/build/ProjectAssets';
+import { Command } from 'commander';
+import fs from 'fs-extra';
+import { uniqBy } from 'lodash';
+import path from 'path';
+
+import { SilentError } from '../CommandError';
+import log from '../log';
+import { Platform } from './eas-build/types';
+import * as CreateApp from './utils/CreateApp';
+
+const bundlePlatforms = [Platform.Android, Platform.iOS];
+
+type Options = {
+  outputDir: string;
+  quiet: boolean;
+  target?: ProjectTarget;
+  force: boolean;
+};
+type PlatformMetadata = { bundle: string; assets: { [key in 'path' | 'ext']: string }[] };
+type FileMetadata = {
+  [key in Platform]: PlatformMetadata;
+};
+type Metadata = {
+  version: number;
```

Comment:
```suggestion
  version: 0;
```
This is the type for metadata v0 anyway? Metadata v1 object might have different fields not included here.

Reply:
great point.